data/json/npcs/NPC_Brigitte_LaCroix.json

```diff
@@ -0,0 +1,343 @@
+        "u_has_var": "talked_to_seer",
+        "type": "dialogue",
+        "context": "first_meeting",
+        "value": "yes",
+        "no": "You there. Quiet down. Can you hear it? The song?",
+        "yes": {
+          "u_has_var": "asked_about_song",
+          "type": "dialogue",
+          "context": "song",
+          "value": "yes",
+          "no": "You're back. Have you come to listen to the song?",
+          "yes": "Acolyte."
+        }
+      },
+	  "speaker_effect": { "effect": { "u_add_var": "talked_to_seer", "type": "dialogue", "context": "first_meeting", "value": "yes" } },
+      "responses": [
+        {
+          "text": "What? What do you mean? What song?",
+          "topic": "TALK_BONE_SEER_SONG",
+          "condition": { "not": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" } },
+          "effect": { "u_add_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+        },
+        {
+          "text": "So about the songs...",
+          "topic": "TALK_BONE_SEER_SONG",
+          "condition": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+        },
+		{
+		"text": "You mentioned some cycle before, when I gave you the bones. What does that mean?",
+		"topic": "TALK_BONE_SEER_CYCLE",
+		"condition": { "u_has_var": "NC_SEER_MISSION_1", "type": "general", "context": "mission", "value": "yes" }
+		},
+        { "text": "Would you like to join me on my travels?", "topic": "TALK_SUGGEST_FOLLOW_SEER", "condition": { "not": "npc_following" } },
+        { "text": "Let's trade items.", "topic": "TALK_BONE_SEER", "effect": "start_trade" },
+        {
+          "text": "Is there a way I can help you with your song?",
+          "topic": "TALK_MISSION_LIST_SEER",
+          "condition": { "u_has_var": "asked_about_song", "type": "dialogue", "context": "song", "value": "yes" }
+        },
+        { "text": "I have to get going. Take care, Seer.", "topic": "TALK_DONE" }
+      ]
+    },
+    {
+      "type": "talk_topic",
+      "id": "TALK_BONE_SEER_SONG",
+      "dynamic_line": "Listen carefully. The bones... they sing. Can you hear it? The song they weave? The stories they hold?",
+      "responses": [
+        { "text": "What? Singing bones? What are you talking about?", "topic": "TALK_BONE_SEER_SONG2" },
+        { "text": "Uh...yeah, sure. I think I'll just be on my way.", "topic": "TALK_DONE" }
+      ]
+    },
+    {
+      "type": "talk_topic",
+      "id": "TALK_BONE_SEER_SONG2",
+      "dynamic_line": "When it all happened, the cataclysm, something... changed. You can see it in all creatures, but most of all their bones. They break, morph, rise again, in an infinite cycle. Living dead walk. Monsters rip and tear each other apart. You can see the resonance, the quiet hum of raw strength, and only by taking the bones does the cycle end - their story, their song, their strength, become yours to use.",
```

Comment:
```suggestion
      "dynamic_line": "When it all happened, <the_cataclysm>, something... changed. You can see it in all creatures, but most of all their bones. They break, morph, rise again, in an infinite cycle. Living dead walk. Monsters rip and tear each other apart. You can see the resonance, the quiet hum of raw strength, and only by taking the bones does the cycle end - their story, their song, their strength, become yours to use.",
```

Reply:
I greatly appreciate all the checking you do. I try to check it mmyself, but sometimes when you write it you sort of fix it unconciously in your brain, plus the keyboard is busted.

Inn this case Cataclysm is actually a stylistic choice. The NPC has a particular way of talking and some of the snippets would not fit her character.

Reply:
Ok, no problems.